lib/hanami/action/mime.rb

```diff
@@ -435,12 +447,33 @@ def accept
         @accept ||= @_env[HTTP_ACCEPT] || DEFAULT_ACCEPT
       end
 
-      # @since 0.1.0
+      # Checks if there is an Accept header for the current request.
+      #
+      # @return [Boolean] true if there's an Accept header to look at
```

Comment:
Fair enough, I don't mind making these changes - just a note though, I did it this was based on other documentation in the same file (https://github.com/Acornsgrow/controller/blame/resolve_content_type_from_custom_formats/lib/hanami/action/mime.rb#L409). 

Should I clean up *all* of the documentation in the file to match?